src/internal/components/LayoutEditor.tsx

```diff
@@ -18,23 +18,18 @@ const devLogger = new DevLogger();
 type LayoutEditorProps = {
   puckConfig: Config;
   templateMetadata: TemplateMetadata;
-  visualConfigurationData: Data;
+  layoutData: Data;
   themeData: ThemeData;
   themeConfig: ThemeConfig | undefined;
 };
 
 export const LayoutEditor = (props: LayoutEditorProps) => {
-  const {
-    puckConfig,
-    templateMetadata,
-    visualConfigurationData,
-    themeData,
-    themeConfig,
-  } = props;
+  const { puckConfig, templateMetadata, layoutData, themeData, themeConfig } =
```

Comment:
Nit, this formatting is a little weird. Should change it to be like how it was before (but with updated names)

Reply:
Yeah I agree I like the old one better but this is what prettier did 🤷‍♂️

Reply:
Oh nvm then